gradoop-flink/src/main/java/org/gradoop/flink/model/api/tpgm/TemporalGraphOperators.java

```diff
@@ -42,6 +50,107 @@
   // Unary Operators
   //----------------------------------------------------------------------------
 
+  /**
+   * Extracts a snapshot of a temporal graph using a given temporal predicate. This will calculate
+   * the subgraph of a temporal graph induced by the predicate.
+   *
+   * The resulting graph is a new logical graph.
+   */
+  default TemporalGraph snapshot(TemporalPredicate predicate) {
+    return callForGraph(new Snapshot(Objects.requireNonNull(predicate)));
+  }
+
+  /**
+   * Extracts a snapshot of a temporal graph using the temporal predicate {@code AS OF t} where
+   * {@code t} is a timestamp in ms as long. This will calculate the subgraph of a temporal graph
+   * induced by the predicate.
+   *
+   * The resulting graph is a new logical graph.
+   *
+   * @see AsOf for further details.
+   */
+  default TemporalGraph asOf(long timestamp) {
+    return snapshot(new AsOf(timestamp));
+  }
+
+  /**
+   * Extracts a snapshot of a temporal graph using the temporal predicate {@code FROM t1 TO t2}
+   * where {@code t1} and {@code t2} are timestamps in ms as long. This will calculate the subgraph
```

Comment:
This might be confusing, since the paramters of this method are not called `t1` and `t2`.